openedx/core/djangoapps/discussions/serializers.py

```diff
@@ -424,7 +424,7 @@ def to_representation(self, instance: CourseDiscussionSettings) -> dict:
             'available_division_schemes': available_division_schemes(course_key),
             'reported_content_email_notifications': instance.reported_content_email_notifications,
             'reported_content_email_notifications_flag':
-                ENABLE_REPORTED_CONTENT_EMAIL_NOTIFICATIONS.is_enabled(course_key),
+                ENABLE_DISCUSSIONS_MFE.is_enabled(course_key),
```

Comment:
I think these attributes should be removed from the backend. Frontend changes will be required for it.
 @asadazam93 @AhtishamShahid any thoughts?